segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -82,6 +84,8 @@ private ServiceBuilder createServiceBuilder() {
 
     public void start() throws Exception {
         Exceptions.checkNotClosed(this.closed, this);
+        // Make sure that the JVM is not caching DNS entries forever.
+        Security.setProperty("networkaddress.cache.ttl", "10");
```

Comment:
I am not sure this would have effect. Probably it should be a command line argument for java

Reply:
Thanks @eolivelli, looking at some documentation I have seen that both methods are similar (for instance https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-jvm-ttl.html). We will look whether this takes effect or not, but in theory it should.